### PR TITLE
Task azure monitor

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
add monitor-lib

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsWorkspace.java [hunk 1]
```diff
@@ -57,4 +61,10 @@ public String getPrimarySharedKeys() {
     public String loadStatus(@Nonnull Workspace remote) {
         return remote.provisioningState().toString();
     }
+
+    public LogsTable executeQuery(String queryString) {
+        LogsQueryOptions options = new LogsQueryOptions().setServerTimeout(Duration.ofSeconds(10));
```
Comment (ivywei0125) [inline azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsWorkspace.java]: NPE check

File: azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsServiceWorkspaceSubscription.java [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
     private final String subscriptionId;
     @Nonnull
     private final LogAnalyticsWorkspaceModule logAnalyticsWorkspaceModule;
+    private final LogsQueryClient logsQueryClient;
 
     protected LogAnalyticsServiceWorkspaceSubscription(@Nonnull String subscriptionId, @Nonnull AzureLogAnalyticsWorkspace service) {
         super(subscriptionId, service);
         this.subscriptionId = subscriptionId;
         this.logAnalyticsWorkspaceModule = new LogAnalyticsWorkspaceModule(this);
+        this.logsQueryClient = new LogsQueryClientBuilder().credential(Azure.az(AzureAccount.class).account().getTokenCredential(subscriptionId)).buildClient();
```
Comment (ivywei0125) [inline azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsServiceWorkspaceSubscription.java:29]: check if nullable

File: azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsWorkspace.java (hunk 1 above)
Comment (ivywei0125) [reply azure-toolkit-libs/azure-toolkit-monitor-lib/src/main/java/com/microsoft/azure/toolkit/lib/monitor/LogAnalyticsWorkspace.java]: getremote() maybe null

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**